site/source/conf.py

```diff
@@ -392,7 +392,8 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
+
```

Comment:
Maybe just delete these lines?  I don't think we need to link to the python standard library anywhere?